server/events/models/models.go

```diff
@@ -443,15 +460,53 @@ type StateRmSuccess struct {
 	RePlanCmd string
 }
 
-// Summary extracts one line summary of policy check.
-func (p *PolicyCheckSuccess) Summary() string {
+func (p *PolicyCheckResults) CombinedOutput() string {
+	combinedOutput := ""
+	for _, psResult := range p.PolicySetResults {
+		// accounting for json output from conftest.
+		for _, psResultLine := range strings.Split(psResult.ConftestOutput, "\\n") {
+			combinedOutput = fmt.Sprintf("%s\n%s", combinedOutput, psResultLine)
```

Comment:
There are two backslashes here on line 467 but only one backslash on line 468. Is that expected?

Reply:
This has to do with conftest output when returning json output, it's needed to accurately measure output length. IIRC the newlines from default output are still parsed correctly.